source/module_io/get_wf_lcao.cpp

```diff
@@ -28,157 +29,142 @@ void IState_Envelope::begin(const psi::Psi<double>* psid,
                             const K_Vectors& kv,
                             const double nelec,
                             const int nbands_istate,
-                            const std::vector<int>& out_band_kb,
+                            const std::vector<int>& out_wfc_norm,
+                            const std::vector<int>& out_wfc_re_im,
                             const int nbands,
                             const int nspin,
                             const int nlocal,
                             const std::string& global_out_dir)
 {
     ModuleBase::TITLE("IState_Envelope", "begin");
 
-    std::cout << " Perform |psi(band, r)| for selected bands." << std::endl;
+    std::cout << " Perform |psi(i, r)|, Re[psi(i, r)], Im[psi(i, r)] for selected bands (gamma only)." << std::endl;
```

Comment:
Although it is not from your code changes, but it seems to be a broken sentence and the meaning is not clear at all.
 "Calculate" instead of "Perform" might be better.